src/pages/login/LoginPage.tsx

```diff
@@ -4,22 +4,29 @@ import classNames from 'classnames';
 import { useCallback } from 'react';
 import { Translation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router';
 import SessionActions from 'src/commons/application/actions/SessionActions';
 import classes from 'src/styles/Login.module.scss';
 
 import Constants from '../../commons/utils/Constants';
 
-const providers = [...Constants.authProviders.entries()].map(([id, { name }]) => ({
-  id,
-  name
-}));
+const providers = [...Constants.authProviders.entries()]
```

Comment:
Would it be better to just use a setting for NUS deployment? In that case we would not need to overcomplicate processing the providers

Reply:
Yeah, makes sense. Will refactor.